packages/turf-isobands/index.js

```diff
@@ -39,7 +35,7 @@ var featureCollection = helpers.featureCollection;
  * //addToMap
  * var addToMap = [isobands];
  */
-module.exports = function (pointGrid, breaks, zProperty, options) {
+export default function (pointGrid, breaks, zProperty, options) {
```

Comment:
@stebogit Same as `@turf/isolines` the test fixtures were modified... no clue why..?

Reply:
😕

Reply:
It's not this line, but the test outputs... it grew in scale by about 40%.

**Before**

https://github.com/Turfjs/turf/blob/321d0df5c77a4005e4a6f28947fe11d463d53bab/packages/turf-isolines/test/out/bigMatrix.geojson

![image](https://user-images.githubusercontent.com/550895/30796121-b62d5886-a19e-11e7-8836-3eaa58509d09.png)


**After**

https://github.com/Turfjs/turf/blob/master/packages/turf-isolines/test/out/bigMatrix.geojson

![image](https://user-images.githubusercontent.com/550895/30796126-bf4a9cb2-a19e-11e7-8f9a-0f51fe121ba5.png)

Reply:
OK, I'll take a look at it; clearly it must be something related to the rescaling part 😅